src/api/anon-email-lists.js

```diff
@@ -25,25 +25,29 @@ function logSubscription({deviceId, email}) {
 	// don't wait on this promise, log in the background
 	call(`/user/${email}`, null, 'GET').then(r => r.json().then(json => {
 		if(r.ok) {
-			logger.info(`anon-email-api subscribed user (device ${deviceId}) as ${json.uuid}`)
+			logger.info(`anon-email-api subscribed user as ${json.uuid}`, { deviceId })
 		} else {
 			const err = new Error(JSON.stringify(json));
 			err.response = r;
 		}
 	})).catch(e => logger.warn(e));
 }
 
-export function subscribe({email, mailingList, deviceId, topics}={}) {
-	logger.info(`anon-email-api about to subscribe user (device ${deviceId}) to ${mailingList} with topics: ${topics}`);
-
+export function subscribe({ email, mailingList, deviceId, topics, following } = { }) {
 	let status;
+	const data = {
+		mailingListName: mailingList,
+		userEmail: email,
+		deviceId
+	};
+	if (following) {
+		data.following = following;
+	} else {
+		data.topics = topics;
+	}
+	logger.info('anon-email-api about to subscribe user', data);
```

Comment:
we might want to be careful about logging the user email here. maybe `n-mask-logger`?